demos/iis-2vm-sql-1vm/azuredeploy.parameters.json

```diff
@@ -10,6 +10,9 @@
     },
     "password": {
       "value": "GEN-PASSWORD"
+    },
+    "modulesUrl": {
```

Comment:
this file needs to be included with the sample and referenced using the artifacts pattern for the repo...

See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

Reply:
referenced using artifacts